### PR TITLE
Minor edits to improve ncov tutorial user experience

### PR DESCRIPTION
**Pull request by huddlej**
### Description of proposed changes

This PR introduces minor edits to improve the user experience of running the new ncov tutorial. See commit messages for more details.

### Related issue(s)

Related to https://github.com/nextstrain/ncov/pull/942

### CODE REVIEW

**Review by victorlin — APPROVED** [review]

**Comment by huddlej** [comment]
Thanks, @victorlin! We can merge this after we merge the PR in ncov with the updated screenshot, etc.

**Pull request merged**
